addons/website_slides/static/tests/tours/slides_tour_tools.js

```diff
@@ -1,5 +1,6 @@
 /** @odoo-module **/
```

Comment:
Hello @pyduf 

Looking good! Thanks for the quick fix on this one.

Can I just ask you to rephrase that part of the commit message?

> We solve that by adding an
“await contains” on the think we check before the check which ensures that what
we are checking is rendered when checked.

"think" -> "thing" ? Maybe use "step" or something more meaningful?
Also, that's a lot of "check" ^^ Could maybe rephrase a bit to make the sentence clearer.

After that we are good to go :rocket: 

Cheers!

Reply:
Hello @awa-odoo , 

Yes indeed, thanks for pointing it out. I have rephrased it.